geomstats/visualization/special_euclidean.py

```diff
@@ -62,3 +73,125 @@ def draw_points(self, ax, points=None, **kwargs):
             color="r",
         )
         ax.scatter(translation[:, 0], translation[:, 1], s=16, **kwargs)
+
+    @staticmethod
+    def plot_geodesic(
```

Comment:
no uppercase letters in python unless they are constant variables (as in the top of this file) or class names.

METRIC=METRIC_SE2 --> metric=METRIC_SE2
N_STEPS=40 --> n_steps=40